src/components/App/App.jsx

```diff
@@ -67,6 +66,8 @@ class App extends React.Component {
     const github = new GitHub()
     github.getCurrentUser().then(user => {
       this.setState({ user })
+      DefaultFilters.init(user.login)
+      this.setState({ filters: Filters.findAll() })
```

Comment:
Sure! It seems weird that this needs to be included two places - post-auth check (if the user is logged in) and post-auth (if they're not). Ideas?